services/ui-src/src/components/export/ExportedReportFieldTable.tsx

```diff
@@ -71,7 +72,17 @@ export const renderGeneralInformation = (
   pageType: string,
   entityType?: string
 ) => {
+  const { report } = useStore() ?? {};
+
+  // check if this was a resubmission
+  const isResubmission =
+    (report?.reportType === "SAR" && report?.submissionCount) ||
+    (report?.status === ReportStatus.SUBMITTED &&
+      report?.submissionCount! > 1 &&
```

Comment:
with this current logic the resubmission question displays upon the first submit, before it's been released.

We have [similar logic](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/blob/main/services/ui-src/src/components/reports/StandardReportPage.tsx#L78-L87) for this in the StandardReportPage component. Maybe it'd make sense to use the same logic so that the question showing up in the form is the same as when it shows up in the PDF?